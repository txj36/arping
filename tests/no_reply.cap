ARP, Request who-has 192.0.2.1 tell 192.0.2.100, length 44
	0x0000:  0001 0800 0604 0001 0001 0233 4400 c000
	0x0010:  0264 0000 0000 0000 c000 0201 0000 0000
	0x0020:  0000 0000 0000 0000 0000 0000
