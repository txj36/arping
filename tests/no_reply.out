ARPING 192[.]0[.]2[.]1
Timeout

--- 192[.]0[.]2[.]1 statistics ---
1 packets transmitted, 0 packets received, 100% unanswered \(0 extra\)
